feature-libs/product/configurators/common/src/core/facade/common-configurator-facade.module.ts

```diff
@@ -0,0 +1,22 @@
+import { ModuleWithProviders, NgModule } from '@angular/core';
+import { ConfiguratorCartService } from './configurator-cart.service';
+import { ConfiguratorCommonsService } from './configurator-commons.service';
+import { ConfiguratorGroupStatusService } from './configurator-group-status.service';
+import { ConfiguratorGroupsService } from './configurator-groups.service';
+import { ConfiguratorFacadeUtilsService } from './utils/configurator-facade-utils.service';
+
+@NgModule({})
+export class CommonConfiguratorFacadeModule {
```

Comment:
I hope we can decorate those services with `@Injectable({providedIn: 'root'})`:
```
        ConfiguratorCommonsService,
        ConfiguratorCartService,
        ConfiguratorGroupsService,
        ConfiguratorFacadeUtilsService,
        ConfiguratorGroupStatusService,
```
This would allow dropping this module at all

btw. Making services `providedIn: 'root'` is a recommended approach.